pkg/taskrunner/copy.go

```diff
@@ -0,0 +1,55 @@
+package taskrunner
+
+import (
+	"fmt"
+	"os"
+
+	cp "github.com/otiai10/copy"
+	"github.com/rs/zerolog/log"
+)
+
+type copyTask struct {
+	Source      string `json:"src"`
+	Destination string `json:"dest"`
+}
+
+func (e *copyTask) exec(c *Config) error {
+	l := log.With().
+		Str("task type", "copy").
+		Str("dest", e.Destination).
+		Logger()
+
+	// Convert source to list of files in case a glob is used
+	source, err := expandPath(c.Root, e.Source)
+	if err != nil {
+		return err
+	}
+
+	// Ensure output directory exists
+	if err := createDirIfNotExists(e.Destination, 0755); err != nil {
+		return err
```

Comment:
Same as above re: error wrapping (although in this one you can get away with it since `createDirIfNotExxists` is your function too)